### PR TITLE
added doc for data transfer to/ from NERC OpenStack VM

### PR DESCRIPTION
**Pull request by Milstein**
Tools documented:

**Generic:**

- [ ] Globus
- [ ] SCP
- [ ] rsync
- [ ] rclone

**GUI based Tools:**

- [ ] WinSCP
- [ ] Cyberduck
- [ ] Filezilla

### CODE REVIEW

File: docs/openstack/advanced-openstack-topics/data-transfer/data-transfer-from-to-vm.md [hunk 1]
```diff
@@ -0,0 +1,568 @@
...
+    ![Save Site WinSCP](images/winscp-save-site.png)
+
+#### Using WinSCP
+
+You can follow above step to manually add a new site next time you open WinSCP
+or, you can connect to your previously saved site (as listed on popup dialog
+will show your all saved site name list) that will show up by just clicking on
+the site name.
+
+Then click "Login" button to connect to your NERC project's VM as shown below:
```
Comment (jtriley) [inline docs/openstack/advanced-openstack-topics/data-transfer/data-transfer-from-to-vm.md:442]: Then click the "Login" button to connect to your NERC project's VM as shown below:

File: docs/openstack/advanced-openstack-topics/data-transfer/data-transfer-from-to-vm.md [hunk 2]
```diff
@@ -0,0 +1,568 @@
...
+
+![Login](images/winscp-site-login.png)
+
+![Successful connection](images/winscp-site-successfully-connected.png)
+
+You should now be connected to the VM's remote directories/files. You can drag
+and drop your files to/from file windows to begin transfer. When done, click the
```
Comment (jtriley) [inline docs/openstack/advanced-openstack-topics/data-transfer/data-transfer-from-to-vm.md:449]: When you're finished, click the "X" icon in the top right to disconnect.

File: docs/openstack/advanced-openstack-topics/data-transfer/data-transfer-from-to-vm.md [hunk 3]
```diff
@@ -0,0 +1,568 @@
...
+X icon up top right to disconnect.
+
+### ii. Cyberduck
+
+[Cyberduck](https://docs.cyberduck.io/cyberduck/) is a libre server and cloud
+storage browser for Mac and Windows. With an easy-to-use interface, connect to
```
Comment (jtriley) [inline docs/openstack/advanced-openstack-topics/data-transfer/data-transfer-from-to-vm.md:455]: Not sure about the "connect to servers" part here - consider rewording or removing

File: docs/openstack/advanced-openstack-topics/data-transfer/data-transfer-from-to-vm.md [hunk 4]
```diff
@@ -0,0 +1,568 @@
...
+servers, enterprise file sharing, and cloud storage.
+
+- **Prerequisites**
+
+- Cyberduck installed, see [Download and Install the latest version of the Cyberduck](https://cyberduck.io/download/)
+for more information.
+
+#### Configuring Cyberduck
+
+- Click on "Open Connection" tab button as shown below:
```
Comment (jtriley) [inline docs/openstack/advanced-openstack-topics/data-transfer/data-transfer-from-to-vm.md:465]: Click on the "Open Connection" button as shown below:

File: docs/openstack/advanced-openstack-topics/data-transfer/data-transfer-from-to-vm.md [hunk 5]
```diff
@@ -0,0 +1,568 @@
...
+        - **all Ubuntu images**: ubuntu
+        - **all CentOS images**: centos
+        - **all Rocky Linux images**: centos
+        - **all Fedora images**: fedora
+        - **all Debian images**: debian
+        - **all RHEL images**: cloud-user
+
+    **"Password"**: "`<Leave blank as you using SSH key>`"
+
+    **"SSH Private Key"**: "Choose the appropriate SSH Private Key from you local
```
Comment (jtriley) [inline docs/openstack/advanced-openstack-topics/data-transfer/data-transfer-from-to-vm.md:491]: from your local machine that has the corresponding public key attached to your VM

File: docs/openstack/advanced-openstack-topics/data-transfer/data-transfer-from-to-vm.md [hunk 6]
```diff
@@ -0,0 +1,568 @@
...
+    machine that has corresponding Public Key attached to your VM"
+
+![Cyberduck SFTP or FTP Configuration](images/cyberduck-open-connection-sftp.png)
+
+#### Using Cyberduck
+
+Then click "Connect" button to connect to your NERC VM as shown below:
```
Comment (jtriley) [inline docs/openstack/advanced-openstack-topics/data-transfer/data-transfer-from-to-vm.md:498]: Then click the "Connect" button to connect to your NERC VM as shown below:

File: docs/openstack/advanced-openstack-topics/data-transfer/data-transfer-from-to-vm.md [hunk 7]
```diff
@@ -0,0 +1,568 @@
...
+
+![Successful connection](images/cyberduck-sftp-successful-connection.png)
+
+You should now be connected to the VM's remote directories/files. You can drag
+and drop your files to/from file windows to begin transfer. When done, click the
```
Comment (jtriley) [inline docs/openstack/advanced-openstack-topics/data-transfer/data-transfer-from-to-vm.md:503]: When you're finished, click the "X" icon in the top right to disconnect.

File: docs/openstack/advanced-openstack-topics/data-transfer/data-transfer-from-to-vm.md [hunk 8]
```diff
@@ -0,0 +1,568 @@
...
+
+#### Using Filezilla
+
+Then click "Connect" button to connect to your NERC VM as shown below:
+
+![Successful connection](images/filezilla-sftp-successful-connection.png)
+
+You should now be connected to the VM and see your local files in the left-hand
+pane and the remote files in the right-hand pane. You can drag and drop between
+them or drag and drop to/from file windows on your computer. When done, click the
```
Comment (jtriley) [inline docs/openstack/advanced-openstack-topics/data-transfer/data-transfer-from-to-vm.md:565]: When you're finished, click the red "X" icon up top to disconnect.

File: docs/openstack/advanced-openstack-topics/data-transfer/data-transfer-from-to-vm.md (hunk 3 above)
Comment (Milstein) [reply docs/openstack/advanced-openstack-topics/data-transfer/data-transfer-from-to-vm.md:455]: Its user-friendly interface enables seamless
connections to servers, enterprise file sharing, and various cloud storage platforms.

**Review by jtriley — APPROVED** [review]

**Pull request merged**